relayer/src/worker/uni_chan_path.rs

```diff
@@ -76,6 +76,11 @@ impl UniChanPathWorker {
                     .into());
                 }
             }
+
+            // If there are no incoming commands, it's safe to backoff.
+            if self.cmd_rx.is_empty() {
+                thread::sleep(Duration::from_millis(200));
+            }
```

Comment:
Maybe we can refactor this by using [`select!` and `after`](https://docs.rs/crossbeam-channel/0.5.1/crossbeam_channel/fn.after.html#examples)? Do you think it would make the code more readable?

ie. we would write something like this instead of `let maybe_cmd = self.cmd_rx.try_recv().ok();`:

```rust
const TIMEOUT = Duration::from_millis(200);

let maybe_cmd = select! {
  recv(self.cmd_rx) -> cmd => Some(cmd),
  recv(after(TIMEOUT)) -> _ => None,
};
```